### DISCUSSION TITLE
Is it possible to load events calendar rotation

### DISCUSSION THREAD

**Discussion by fsajbert** [vkurko/calendar#180] [Q&A]
Hi,
I'm planning to have number of events each week, so thinking about possibility to initially load events for the current week/day and then when you navigate to next week, load new bunch of events. Is it possible to implement such behaviour? 
Thanks

**Comment by vkurko** [comment]
Hello. Sure, it is possible. Please take a look at the [eventSources](https://github.com/vkurko/calendar#eventsources) option.
